pkg/controller/csr_check.go

```diff
@@ -77,25 +77,38 @@ func validateCSRContents(req *certificatesv1.CertificateSigningRequest, csr *x50
 		return "", fmt.Errorf("%q not in %q and %q", groupSet, "system:authenticated", nodeGroup)
 	}
 
+	var validationUsageSet []string
+
+	switch csr.PublicKeyAlgorithm {
+	// ECDSA keys only require UsageDigitalSignature and UsageServerAuth
+	case x509.ECDSA:
```

Comment:
In the renewal controller upstream they are allowing both usage sets for ECDSA, which I think is for upgrades, we will need to work out a way to refactor this to allow both as this will break 4.13 compat as it is right now

Reply:
Fixed